src/libstd/keyword_docs.rs

```diff
@@ -603,9 +633,58 @@ mod let_keyword { }
 /// [Reference]: ../reference/expressions/loop-expr.html
 mod loop_keyword { }
 
+#[doc(keyword = "match")]
+//
+/// Control flow based on pattern matching.
+mod match_keyword { }
+
+#[doc(keyword = "mod")]
+//
+/// Organize code into [modules].
+///
+/// [modules]: ../../nightly/book/ch07-02-defining-modules-to-control-scope-and-privacy.html
+mod mod_keyword { }
+
+#[doc(keyword = "move")]
+//
+/// Capture a [closure]'s environment by value.
+///
+/// [closure]: ../book/second-edition/ch13-01-closures.html
+mod move_keyword { }
+
+#[doc(keyword = "mut")]
+//
+/// A mutable binding, reference or pointer.
+mod mut_keyword { }
+
+#[doc(keyword = "pub")]
+//
+/// Make an item visible to others.
```

Comment:
This isn't restricted to items, e.g. fields may also be `pub`.

Reply:
Is there a better word than "something"? I know "item" has a specific meaning in Rust's grammar.

Reply:
cc @petrochenkov

Reply:
Seems ok for a summary.

Reply:
Fair enough.